tests/garage/tf/models/test_gaussian_mlp_model.py

```diff
@@ -11,99 +12,105 @@ def setUp(self):
         super().setUp()
         self.input_var = tf.placeholder(tf.float32, shape=(None, 5))
         self.obs = np.ones((1, 5))
+        self.output_dim = 2
 
-    def test_gaussian_mlp_model_with_std_share_network(self):
-        model = GaussianMLPModel(output_dim=2, std_share_network=True)
+    def test_std_share_network(self):
+        model = GaussianMLPModel(
+            output_dim=self.output_dim, std_share_network=True)
         outputs = model.build(self.input_var)
         out, model_out = self.sess.run(
             [outputs[:-1], model.networks['default'].outputs[:-1]],
             feed_dict={model.networks['default'].input: self.obs})
 
         assert np.array_equal(out, model_out)
-        # output shape should be 2 * output_dim
+
+        # --- output shape ---
```

Comment:
if you find yourself creating comment blocks in your tests like this, your test is too long.

please split your tests into more test methods.